operations/host_spawn.go

```diff
@@ -270,49 +291,83 @@ func hostConfigure() cli.Command {
 			if err != nil {
 				return errors.Wrapf(err, "can't find project for queue id '%s'", project)
 			}
-			cmds, err := getJasperCommands(projectRef, directory, quiet)
+
+			if distroName != "" {
+				distro, err := client.GetDistroByName(ctx, distroName)
+				if err != nil {
+					return errors.Wrap(err, "problem getting distro")
+				}
+
+				if distro.IsVirtualWorkstation {
```

Comment:
Do they _have_ to specify distro name for virtual workstations to setup correctly then? (Without specifying a distro we'll miss this step and they won't have directory built correctly, right?) Perhaps we should actually restrict this command to workstations then and make the distro flag required (or at least required if directory is specified). 

Also probably add distro name to the host page? Will users definitely remember what distro a host is?

Reply:
I'm not sure, I think we should probably iterate on this. 

I imagine a future world where you click buttons on the UI to get the app to run this command via jasper, which means people might not type it in many cases? In cases where you've specified your own directory (or are using the default current directory,) I think it's fine to just put it where you tell us (and this makes it work "predictably" on people's laptops. 

I'm clarifying the help text.